Dockerfile.build

```diff
@@ -1,2 +1,2 @@
 FROM golang:1.8.3
-RUN curl -s -o - https://get.docker.com/builds/Linux/x86_64/docker-1.9.1 > /usr/local/bin/docker && chmod +x /usr/local/bin/docker
+RUN curl -sSL https://get.docker.com | sh
```

Comment:
would like this pinned to a known version

Reply:
Don't think it can be pinned when using the convenience script, though.

Reply:
That may be true. However, we'll still need to find a way to pin the version using some other method.

Reply:
Maybe create our own convenience script based on that one, i'll see what I can do